### ISSUE TITLE
Task to update the versions one by one

### ISSUE THREAD

**Issue by daviddenton** [jmfayard/gradle-kotlin-dsl-libs#1]
Thanks for the PR on http4k - looks like it will be super useful. But now that I'm manually updating (some) dependencies, I'd really like a way in which I could update them in an interactive way without manually copying the strings.

I'd imagine running a gradle task which asks me (one-by-one) if I'd like to upgrade an out-of-date dependency from the old version to the new. Maybe with an option to just "accept all updates". This would then regenerate the source file containing the versions.

**Comment by jmfayard** [comment]
I'm glad you like it @daviddenton  thanks for your kind words! 

I don't think this task of updating the versions should be left to a software. 
I did want to automate what it is the latest version because that part is super boring. 
But as an Android developer, I had my share of Android Studio automatically updating my build, 
and the negatives of when it doesn't work are vastly superior to the benefits for when it works. 
For example it's usually a good idea to read the `CHANGELOG.md` before updating (and that's why I added a link to the website in the generated code!)

Your thoughts on that? 

Maybe it's enough to make it just a bit convenient by having the code like this

![image](https://user-images.githubusercontent.com/459464/46918101-004c1980-cfce-11e8-84c9-eed4e8faadc2.png)

Select the same part as me, press delete, and that's it, you have updated the dependency!
